packages/apollo-codegen-swift/src/language.ts

```diff
@@ -29,7 +29,16 @@ export interface Property {
 }
 
 export function escapedString(string: string) {
-  return string.replace(/"/g, '\\"').replace(/\n/g, "\\n");
+  var lines = string
+    .split(/\n/g)
+    .map(line => {
+      return line.trim();
```

Comment:
This is going to trim meaningful whitespace from multiline strings. As an example from the spec:

```graphql
mutation {
  sendEmail(message: """
    Hello,
      World!

    Yours,
      GraphQL.
  """)
}
```

is equivalent to

```graphql
mutation {
  sendEmail(message: "Hello,\n  World!\n\nYours,\n  GraphQL.")
}
```

but with this PR it would instead turn into

```graphql
mutation {
  sendEmail(message: "Hello,\nWorld!\n\nYours,\nGraphQL.")
}
```